docker/pegasus-build-env/centos6/Dockerfile

```diff
@@ -76,12 +76,6 @@ ENV MAVEN_HOME=/opt/rh/rh-maven33/root/usr/
 ENV PATH=$MAVEN_HOME/bin:$JAVA_HOME/bin:$GCC_HOME/bin:$PYTHON2_HOME/bin/:$PATH
 ENV LD_LIBRARY_PATH=$JAVA_HOME/jre/lib/amd64/server:$PYTHON2_HOME/lib64/:$LD_LIBRARY_PATH
 
-RUN pip install --no-cache-dir cmake
-
-RUN wget --progress=dot:giga https://github.com/apache/thrift/archive/refs/tags/0.11.0.tar.gz -P /opt/thrift && \
```

Comment:
thrift is used to generate cpp/java code from IDL, why removed it?

Reply:
This patch added these code: https://github.com/apache/incubator-pegasus/pull/1045

That's because cpp code link thrift in version 0.9.3 (build by thirdparty), java code use thrift 0.11.0 binary (download here) to generate code from *.thrift files. Although it's confused, we can improve the different version lib/bin in another patch, but it would better to keep the logic in this patch.